### PR TITLE
[RHDEVDOCS-2333] Update ability to filter helm charts when multiple Helm repos are available

### PR DESCRIPTION
**Pull request by rishumehra**
Updated the information for Chart Repositories filter with its screenshot and a few minor changes.

JIRA issue: [RHDEVDOCS-2333]  | URL: https://issues.redhat.com/browse/RHDEVDOCS-2333

Version: OpenShift 4.7

### CODE REVIEW

**Review by rishumehra — COMMENTED** [review]
Please check the changes

File: modules/helm-adding-helm-chart-repositories.adoc [hunk 1]
```diff
@@ -43,3 +43,13 @@ EOF
...
 . Navigate to  the *Developer Catalog* in the web console to verify that the helm charts from the Azure chart repository are displayed.
++
+Use the *Chart Repositories* filter to search for the required Helm Chart
++
+.Chart Repositories filter
+image::helm-chart-repo-filer.png[]
++
+[NOTE]
+====
+If an admin removes all the chart repos, the user cannot view the ability to add or view Helm Charts from the navigation or forms.
```
Comment (Preeticp) [inline modules/helm-adding-helm-chart-repositories.adoc:54]: We will need to make this note more appropriate for the docs, and can not use it as is:

s/an admin/a cluster administrator
s/cannot view the ability to add or view Helm Charts from the navigation or forms./ cannot view the Helm option in the *+Add* view, *Developer Catalog*, and left navigation panel (Please test and validate that this is correct.) and thus cannot add Helm Charts.

Comment (Preeticp) [reply modules/helm-adding-helm-chart-repositories.adoc:54]: This is not part of this PR, but can you please make this change too on Line 8:

s/You can add/As a cluster administrator, you can add...

**Review by Preeticp — CHANGES_REQUESTED** [review]
@rishumehra looks good, I have added a few comments, PTAL.

File: modules/helm-adding-helm-chart-repositories.adoc (hunk 1 above)
Comment (rishumehra) [reply modules/helm-adding-helm-chart-repositories.adoc]: Fixed the comment in the new commit.

File: modules/helm-adding-helm-chart-repositories.adoc [hunk 2]
```diff
@@ -43,3 +43,13 @@ EOF
 ----
 +
 . Navigate to  the *Developer Catalog* in the web console to verify that the helm charts from the Azure chart repository are displayed.
++
+Use the *Chart Repositories* filter to search for the required Helm Chart
++
+.Chart Repositories filter
+image::helm-chart-repo-filer.png[]
```
[the remark this answers is not in the document]
Comment (rishumehra) [reply modules/helm-adding-helm-chart-repositories.adoc]: Fixed it.

**Comment by rishumehra** [comment]
Closing this one and opening a new PR

**Pull request closed without merging**